web/app.php

```diff
@@ -2,7 +2,7 @@
 
 use Symfony\Component\HttpFoundation\Request;
 
-/**
+/*
```

Comment:
i was surprised by styleci. maybe its because its not inside a class but in a global php file. what should we do? styleci will go red if i add the `*` back.